content-src/components/DiscoveryStreamComponents/DSFooter/DSFooter.jsx

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { cardContextTypes } from "../../Card/types.js";
+import React from "react";
+
+const StatusMessage = ({ icon, fluentID }) => (
+  <div className="status-message">
+    <span
+      aria-haspopup="true"
+      className={`story-badge-icon icon icon-${icon}`}
+    />
+    <div className="story-context-label" data-l10n-id={fluentID} />
+  </div>
+);
+
+export class DSFooter extends React.PureComponent {
```

Comment:
How about renaming DSFooter component to something which is not tied to placement and more with purpose something like ContextMessage

Reply:
I'm wary of tying it to a particular usecase at this stage since we're still experimenting. I think `ds-story-footer` is fine for now.